src/dns-operation/dnsResolver.js

```diff
@@ -24,11 +24,14 @@ export default class DNSResolver {
     if (envutil.isNode() && !this.http2) {
       this.http2 = await import("http2");
       this.nodeUtil = await import("../helpers/node/util.js");
+      log.i("create custom http2 client");
```

Comment:
When looking at logs, capitalization of first letter helps in identifying the start of it.

Reply:
Usually, `tag` identifies a log message (and it's beginning too). I'll add them in a follow-up commit tweaking the logging layer a bit, to add request `trace`s and `tags`.